core/src/components/textarea/textarea.tsx

```diff
@@ -159,16 +159,23 @@ export class Textarea implements ComponentInterface {
     }
     this.runAutoGrow();
     this.emitStyle();
-    this.ionChange.emit({ value });
   }
 
   /**
-   * Emitted when the input value has changed.
+   * The `ionChange` event is fired for `<ion-textarea>` elements when the user
+   * modifies the element's value. Unlike the `ionInput` event, the `ionChange`
+   * event is not necessarily fired for each alteration to an element's value.
+   *
+   * Depending on the way the users interacts with the element, the `ionChange`
+   * event fires at a different moment:
+   * - When the element loses focus after its value has changed : for elements
+   * where the user's interaction is typing.
```

Comment:
Could you rephrase this somehow? The formatting looks a bit odd and I'm not sure I understand what it's saying 🤔

Reply:
Do you have an example of what would be more preferred? This was inspired from the Mozilla docs around the `change` event: https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/change_event

With `ion-textarea` not having extra conditions for when `ionChange` fires however, I could see possibly something like:
> The `ionChange` event is fired when the element loses focus after its value has changed.

Thoughts?

Reply:
Yeah, that sounds good to me. The current description is strange because it mentions the event firing at different moments, but then only one condition is listed. Said condition also seems to include elements besides `textarea`, which isn't needed since this is explicitly the `ion-textarea` docs.